src/inverted_index.c

```diff
@@ -341,13 +341,8 @@ ENCODER(encodeNumeric) {
   } else {
     // Floating point
     NumEncodingFloat *encFloat = &header.encFloat;
-    if (fabs(absVal - f32Num) < 0.01) {
```

Comment:
Why not do:
`if (fabs(absVal - f32Num) == 0)`
?
What am I missing?

Reply:
Maybe even make it configurable so we can easily revert if an issue will be raised?

Reply:
You are right, we can maintain it though it will have an effect only in rare cases when numbers are 1/2, 1/4, 1/8 etc'
Since we keep in the reader anyway, why not!

@K-Jo what do you think about making it configurable?

Reply:
@MeirShpilraien done